rosbag2_py/src/rosbag2_py/_transport.cpp

```diff
@@ -356,7 +357,11 @@ PYBIND11_MODULE(_transport, m) {
 
   py::class_<rosbag2_py::Recorder>(m, "Recorder")
   .def(py::init())
-  .def("record", &rosbag2_py::Recorder::record)
+  .def(
+    "record", &rosbag2_py::Recorder::record,
+    py::arg("storage_options"),
+    py::arg("record_options"),
+    py::arg("node_name") = "rosbag2_recorder")
```

Comment:
This could be defaulted above in the C++ function definition, and then you can skip specifying args here, save a few lines.

Reply:
I don't really have much experience with Pybind11, tried searching online, I made a change here but I don't think this is what was expected, do you have somewhere to read on this topic? As for the CLI parameter I already implemented the feature as well, Thanks a lot for the help!

Reply:
No worries, I realize actually that my defaulting suggestion is not good. But I would suggest adding argument names, I've added that as an inline suggestion above